_builder_lib/docsitebuilder/helpers.rb

```diff
@@ -15,7 +15,19 @@ module Helpers
     PACKAGE_DIRNAME     = '_package'
     BLANK_STRING_RE     = Regexp.new('^\s*$')
     PRODUCT_AUTHOR      = "OpenShift Documentation Project <dev@lists.openshift.redhat.com>"
-    ANALYTICS_SHIM      = '<script type="text/javascript" src="https://assets.openshift.net/app/assets/site/tracking.js"></script>'
+    ANALYTICS_SHIM      = {
+      'OpenShift Origin' => "<script async src=\"//www.google-analytics.com/analytics.js\" type=\"text/javascript\"></script>
```

Comment:
Sounds good, that's how I had it written originally until I realized I didn't have access to the key from the page itself :)